src/options.rs

```diff
@@ -10,6 +10,7 @@ use clap::ArgMatches;
 use layout_options::*;
 use regex::Regex;
 use serde::{Deserialize, Serialize};
+use starship_battery::Manager;
```

Comment:
This fails if the battery features is not turned on, please guard it with a `#cfg` directive.

Reply:
I added the guarding using another account unfortunately (NitroDev), which is another account of mine. It should be all working fine now.